src/plugins/crypto/README.md

```diff
@@ -28,6 +52,8 @@ The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
 The key derivation is still WIP.
 
+The status of this plugin is experimental. Use it at your own risk!
```

Comment:
Its better to use the to sentence "The status of this plugin is experimental." at the start of the `Restrictions` section.

The usage is always on one's own risk, the question is what a user can expect ;)